helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroCompatibilityHelper.java

```diff
@@ -14,18 +14,6 @@
 import com.linkedin.avroutil1.compatibility.avro17.Avro17Adapter;
 import com.linkedin.avroutil1.compatibility.avro18.Avro18Adapter;
 import com.linkedin.avroutil1.compatibility.avro19.Avro19Adapter;
-import java.io.ByteArrayInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.ObjectInput;
-import java.io.ObjectOutput;
-import java.io.OutputStream;
-import java.lang.reflect.Field;
-import java.lang.reflect.Modifier;
-import java.nio.charset.StandardCharsets;
-import java.util.Collection;
-import java.util.List;
```

Comment:
It looks like this was a `Cmd+Z` from when I Optimized Imports on this file. Do you organize imports by hand or use different settings?

Reply:
i let intellij do it, do you have custom settings on your IDE ?